src/dev.html

```diff
@@ -18,6 +18,8 @@
 
     <header id="site-header"></header>
 
+    <mkt-nav role="navigation" id="site-nav"></mkt-nav>
```

Comment:
It's not problematic nor inflexible nor messy. We inject *everything* via JS; our apps are all single-page apps. What's messy is cluttering our base index.html file with elements that projects may or may not use. And you have to touch Zamboni whenever you want to update it.